pkg/syncer/status/status_process_test.go

```diff
@@ -383,7 +383,9 @@ func TestSyncerProcess(t *testing.T) {
 			workloadClusterName:                 "us-west1",
 
 			expectActionsOnFrom: []clienttesting.Action{},
-			expectActionsOnTo:   []clienttesting.Action{},
+			expectActionsOnTo: []clienttesting.Action{
+				getDeploymentAction("theDeployment", "test"),
```

Comment:
Why is this action expected now (that wasn't before) ?

Reply:
As we removed some of the functions that were gated before: https://github.com/kcp-dev/kcp/pull/1228/files#diff-a4fd3bdfe5e149994aadff9dd072c7cbe8975e673dfcd4c2b7eba60f3c131482L103

Now the EnsureUpstreamFinalizerRemoved is called, doing a `Get` of the object.

Reply:
Ah, yes. Thanks for the detailed explanation that reminds me something !